src/mudlet.h

```diff
@@ -137,6 +137,13 @@ class mudlet : public QMainWindow, public Ui::main_window
     mudlet();
     ~mudlet() override;
 
+    enum class BuildType {
+        Unknown = 0,
+        NotUpdateable, // Previously developmentVersion = true
+        Release, // Previously releaseVersion = true
+        PublicTest // Previously publicTestVersion = true
+    };
```

Comment:
Going forward I think this would be more helpful for anyone working with the codebase:

```suggestion
    enum class BuildType {
      Unknown = 0,          // Uninitialized state (should never be used)
      NotUpdateable,        // Development or distribution package build
      Release,              // Official release build with auto-updater
      PublicTest            // Public test build (PTB) with auto-updater
  };
```